test/fixtures/valid-code

```diff
@@ -64,11 +64,14 @@ function plugin(opts) {
   }
 }
 
+const noArrowFuncShorthand = (a) => { return a }
```

Comment:
There is a rule for this too - function expressions which we've been removing in favor of function delcarations

Reply:
https://eslint.org/docs/rules/func-style

we should likely set this to declaration

Reply:
What's the rule name?  Will it discern when we want anonymous functions (I suppose it could).